imports/plugins/included/ui-search/client/templates/searchModal/searchModal.js

```diff
@@ -43,10 +44,60 @@ Template.searchModal.onCreated(function () {
     }
   });
 
+    // Sort products by price
+  const sort = (products, type) => {
+    return products.sort((a, b) => {
+      const A = a.price === null ? -1 : a.price.min;
+      const B = b.price === null ? -1 : b.price.min;
+      if (A < B) {
+        return type === "DESC" ? 1 : -1;
+      } else if (A > B) {
+        return type === "ASC" ? 1 : -1;
+      }
+      return 0;
+    });
+  };
+
+ // filter product by price
+  const filterProductByPrice = (products, query) => {
+    return _.filter(products, (product) => {
+      if (product.price) {
+        const maxPrice = parseFloat(product.price.max);
+        const minPrice = parseFloat(product.price.min);
+        const queryMaxPrice = parseFloat(query[1]);
+        const queryMinPrice = parseFloat(query[0]);
+        if (minPrice >= queryMinPrice && maxPrice <= queryMaxPrice) {
```

Comment:
you can just return this line here `minPrice >= queryMinPrice && maxPrice <= queryMaxPrice`
filter expects a true/false return